### PR TITLE
Updated tests for UiUx page

### PR DESCRIPTION
**Pull request by Valerii-Kryvokhyzha**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignContent.test.ts [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+	await expect(successStoriesContainer.getByTestId(MainSiteButtons.ReadFullCaseStudies)).toHaveText(
+		'Read Full Case Studies'
+	);
+});
+
+test('Check carousel section numbers and titles in "Typical UX/UI Design Workflow" container from the "UX/UI Design" page @Regression @UxUiDesign @TSWEB-670', async () => {
+	const typicalUxUiDesignWorkflowContainer = driver.getByTestId(UxUiDesign.TypicalUxUiDesignWorkflow);
+	const carousel = typicalUxUiDesignWorkflowContainer.getByTestId(Container.ContainerCarousel);
+
+	await expect(carousel.getByTestId(Container.SectionNumber)).toHaveText([
```
Comment (ysukhova) [inline ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignContent.test.ts:130]: What about shortening 130-159 lines in the following way:
```
for(let i = 1; i <= 11; i++) {
		const sectionNumber = i.toString().padStart(2, '0');
		await expect(carousel.getByTestId(Container.SectionNumber).nth(i-1)).toHaveText(sectionNumber);
	}

	await expect(carousel.getByTestId(Container.SectionTitle)).toHaveText([
		'Business requirements allocation',
		'Market and business domain analysis',
...]);
```

File: ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignActions.test.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+		[linkedInButtons.nth(2), ExpertsLinkedInLinks.ElizabethMalygina],
+		[linkedInButtons.nth(3), ExpertsLinkedInLinks.HannaZhyhan],
+		[linkedInButtons.nth(4), ExpertsLinkedInLinks.YelyzavetaLvova],
+	]);
+
+	await baseDriverSteps.checkRedirectToPages(buttonUrlMap);
+});
+
+// Unskip after Blog will be stable
+test.skip('Check redirect by Blog button in "We Never Stop Improving Your Product" container from the "UX/UI Design" page @Regression @UxUiDesign @TSWEB-670', async () => {
```
Comment (ysukhova) [inline ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignActions.test.ts:77]: Please add jiraId for skip

**Review by ysukhova — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignActions.test.ts [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+import ExpertsBehanceLinks from '../../../../../preconditionsData/Links/ExpertsBehanceLinks';
+import Container from '../../../../../identifiers/Container';
+
+test.beforeEach(async () => {
+	await baseDriverSteps.createsNewBrowserAndGoToUrl(UrlProvider.urlBuilder(UrlPath.UiUxDesign));
+});
+
+test('Check redirect to clutch in "Success Stories" container from the "UX/UI Design" page @Regression @UxUiDesign @TSWEB-670', async () => {
+	const successStoriesContainer = driver.getByTestId(UxUiDesign.SuccessStories);
+	const clutchButton = successStoriesContainer.getByTestId(Buttons.Clutch);
```
Comment (UncleDanya) [inline ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignActions.test.ts:25]: Why do you declare a variable here if you only click on it next? You can click on it in one line.

File: ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignActions.test.ts [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+	const newPage = await driver.DriverContext.waitForEvent('page');
+	expect(newPage.url()).toEqual(
+		UrlProvider.urlBuilder(`${UrlPath.AuthorPage}${AuthorsEnum.DmytroDytiuk}`, Environment.Production)
+	);
+	await newPage.close();
+});
+
+test('Check redirect by Behance button in "We Never Stop Improving Your Product" container from the "UX/UI Design" page @Regression @UxUiDesign @TSWEB-670', async () => {
+	const weNeverStopImprovingContainer = driver.getByTestId(UxUiDesign.WeNeverStopImprovingYourProduct);
+	const behanceButton = weNeverStopImprovingContainer.getByTestId(Buttons.Behance);
```
Comment (UncleDanya) [inline ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignActions.test.ts:90]: same

File: ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignContent.test.ts [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+	await expect(successStoriesContainer.getByTestId(MainSiteButtons.ReadFullCaseStudies)).toHaveText(
+		'Read Full Case Studies'
+	);
+});
+
+test('Check carousel section numbers and titles in "Typical UX/UI Design Workflow" container from the "UX/UI Design" page @Regression @UxUiDesign @TSWEB-670', async () => {
+	const typicalUxUiDesignWorkflowContainer = driver.getByTestId(UxUiDesign.TypicalUxUiDesignWorkflow);
+	const carousel = typicalUxUiDesignWorkflowContainer.getByTestId(Container.ContainerCarousel);
+
+	for (let i = 1; i <= 11; i++) {
```
Comment (UncleDanya) [inline ui_tests_playwright/automation/test/ui/MainSite/services/UxUiDesign/UxUiDesignContent.test.ts:130]: Why you use hardcode here?

**Review by ysukhova — APPROVED** [review]

**Review by anastasiia-liashova — APPROVED** [review]

**Review by UncleDanya — APPROVED** [review]

**Pull request merged**